frontend/src/helpers/Colors.js

```diff
@@ -1,7 +1,20 @@
-import COLORS from '../constants/Colors.js';
+import COLORS, { COLOR_NOT_FOUND } from '../constants/Colors.js';
+
+const indexOf = (string, query) => {
```

Comment:
please rename the parameters to be more descriptive thanks!

Reply:
fixed